src/main/java/usecases/CourseEvaluator/CheckAnswer.java

```diff
@@ -0,0 +1,23 @@
+package usecases.CourseEvaluator;
+import entities.Course;
+
+
+public class CheckAnswer {
+
+    public int compare(Course student, Course original) {
+
+        int counter = 0;
+
+        for (int i = 0; i < student.getQuestions().size(); i++){
+            if (student.getQuestionAt(i).getAnswer().equals(original.getQuestionAt(i).getAnswer())){
+                counter += original.getQuestionAt(i).getPoints();
+            }
```

Comment:
Perfect ! This is exactly how we originally planned to check the answers. I have a slight suggestion which is up to you to decide.

Maybe if you lowercase both answers (by using a built-in method that lowercases Strings) we could compare a greater number of matching Strings !

So if we were to compare
True and true
They would successfully compare ! Whereas in the current implementation the above comparison would return false (even though logically it should be true) Up to you to make this change.